handlers/v1/countries/countries.go

```diff
@@ -0,0 +1,12 @@
+package countries
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/jegacs/golang-bootcamp-2020/handlers/v1/countries/handlers"
+)
+
+func Countries(group *gin.RouterGroup) {
```

Comment:
By convention, all the descriptions should start with the name of the struct, interface, function, etc name